apps/judicial-system/web/src/components/InfoCard/VictimInfo/VictimInfo.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC } from 'react'
+
+import { Box, Text } from '@island.is/island-ui/core'
+import { formatDOB } from '@island.is/judicial-system/formatters'
+import { Victim } from '@island.is/judicial-system-web/src/graphql/schema'
+
+import RenderPersonalData from '../RenderPersonalInfo/RenderPersonalInfo'
+
+interface VictimInfoProps {
+  victim: Victim
+}
+
+export const VictimInfo: FC<VictimInfoProps> = (props) => {
+  const { victim } = props
+
+  return (
+    <>
+      <Box marginBottom={1}>
```

Comment:
Consider adding a `component="p"` to this Box so that this is a paragraph, not just two individual spans. ☺️